lib/test.d.ts

```diff
@@ -1,25 +1,31 @@
 interface TestOptions {
   /**
-   * The number of tests that can be run at the same time. If unspecified, subtests inherit this value from their parent.
-   * Default: 1.
+   * If a number is provided, then that many tests would run in parallel. If truthy, it would run (number of cpu cores - 1) tests in parallel. For subtests, it will be Infinity tests in parallel. If falsy, it would only run one test at a time. If unspecified, subtests inherit this value from their parent.
```

Comment:
```suggestion
   * If a number is provided, then that many tests would run in parallel. If truthy, it
   * would run (number of cpu cores - 1) tests in parallel. For subtests, it will be
   * `Infinity` tests in parallel. If falsy, it would only run one test at a time. If
   * unspecified, subtests inherit this value from their parent.
```

Reply:
@aduh95 good idea, I've formatted the entire file with prettier + prettier-plugin-jsdoc, see https://github.com/nodejs/node-core-test/pull/42/commits/29c3de0a8fa89ddbfcaee7dc71f8a2a28c3e1e63